partials/schema-prop.html

```diff
@@ -8,6 +8,10 @@
     <div class="{% if prop | isExpandable %}js-prop cursor-pointer py-2{% endif %} flex property">
       <div class="pr-4" style="margin-top:-2px; min-width: 25%;">
         <span class="text-sm{% if specialName %} italic text-grey{% endif %}{% if prop.deprecated() %} line-through{% endif %}" style="word-break: break-word;">{{propName}}</span>
+        {# Making sure that only human readable uid is rendered, instead of the anonymous ones that start with < #}
+        {% if not prop.uid() | startsWith('<') %}
+          <span class="border text-orange rounded text-xs ml-3 py-0 px-2">{{prop.uid()}}</span>
+        {% endif %}
```

Comment:
by adding this we nicely display uid of the schema

now it looks like 
<img width="942" alt="Screenshot 2020-10-15 at 17 10 43" src="https://user-images.githubusercontent.com/6995927/96149745-f5545880-0f09-11eb-8880-7335496cdc44.png">

and before it was

<img width="942" alt="Screenshot 2020-10-15 at 17 15 20" src="https://user-images.githubusercontent.com/6995927/96149799-04d3a180-0f0a-11eb-9668-f706de8202ee.png">

using such a schema
```
asyncapi: 2.0.0
info:
  title: Test MQTT API
  version: '1.0.0'
  description: |
    Test


defaultContentType: application/json

channels:  
  foo/bar:
    description: Test MQTT operation
    publish:

      message:
        $ref: '#/components/messages/testMessage'
components:
  messages:
    testMessage:
      contentType: application/json
      payload:
        $ref: "#/components/schemas/mainSchema"
  schemas:
 
    oneOfSchema:
      type: object
      properties:
        foo:
          type: number
          format: float	
    oneOfSchema2:
      type: object
      properties:
        foo:
          type: number
          format: float	    

             
    mainSchema:
      type: object
      properties:
        mixedTypeArray:
          type: array
          items:
            oneOf:
            - $ref: "#/components/schemas/oneOfSchema"
            - $ref: "#/components/schemas/oneOfSchema2"
            - type: object
              properties:
                foo:
                  type: number
                  format: float	  
```

Reply:
Is it important for the end-user? I mean, the user consuming the documentation and that may not have a clue this is using AsyncAPI.

Reply:
@fmvilas yes, the motivation came from 2 users here https://github.com/asyncapi/generator/issues/427. they saw it in OpenAPI generation

Reply:
Yes its important to the reader to know what they can and can't use.  I second adding this, its an issue for me.

Reply:
I see 👍